fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -1534,8 +1534,7 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueue* self,
 						    rd.healthPriority == SERVER_KNOBS->PRIORITY_TEAM_0_LEFT)
 							inflightPenalty = SERVER_KNOBS->INFLIGHT_PENALTY_ONE_LEFT;
 
-						auto req = GetTeamRequest(WantNewServers(rd.wantsNewServers),
-						                          wantTrueBest,
+						auto req = GetTeamRequest(TeamSelect::AVOIDMOVEMENT,
```

Comment:
The `TeamSelect` value here should depend on whether `rd.wantsNewServers` is true/false to avoid any behavior change. (Also see related comment below in `DDTeamCollection.actor.cpp`.)

Reply:
Yes, we should probably not introduce any new behavior in this commit. Just refactor GetTeamRequest